pom.xml

```diff
@@ -18,7 +18,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <guava.version>11.0.2</guava.version>
+        <guava.version>19.0</guava.version>
```

Comment:
The problem is that the old one is not OSGi ready, did you get compatibility issues after 11.x ?